ui/redux/reducers/collections.js

```diff
@@ -3,35 +3,37 @@ import { handleActions } from 'util/redux-utils';
 import * as ACTIONS from 'constants/action_types';
 import * as COLS from 'constants/collections';
 
-const getTimestamp = () => {
-  return Math.floor(Date.now() / 1000);
-};
-
```

Comment:
:exclamation: Changing to just `Date.now()` creates a discrepancy with LBRY Desktop.  The timestamp changes between seconds and milliseconds as I switch from Desktop to Odysee, since this is synced to the wallet.

Reply:
I think the wallet data uses seconds for `updatedAt`.  Would changing this to ms causes the sync merge to break?

Reply:
for some reason when I use the previous function here `/ 1000` it returns 1970 as date

Reply:
Is it something local to the machine? https://stackoverflow.com/questions/26642990/javascript-s-date-now-returning-0-zero.